### PR TITLE
update metadata + add social card

### PR DESCRIPTION
**Pull request by friendlymatthew**
Ticket: https://github.com/twentyhq/twenty/issues/1379

Screen Shot 2023-08-30 at 2 16 18 PM

### CODE REVIEW

**Review by FelixMalfait — APPROVED** [review]
That was fast! Looks good to me

File: front/index.html [hunk 1]
```diff
@@ -5,17 +5,33 @@
     <link rel="icon" href="/favicon.ico" />
```
Comment (charlesBochet) [inline front/index.html:5]: I think this file is not used actually. Can you make all your changes in front/public/index.html and remove this front/index.html file?

**Review by charlesBochet — CHANGES_REQUESTED** [review]

File: front/index.html (hunk 1 above)
Comment (friendlymatthew) [reply front/index.html:5]: Yeah - haha was wondering why there were two index.htmls. Will do!

**Review by charlesBochet — APPROVED** [review]

**Comment by charlesBochet** [comment]
Thank you, merging it!

**Pull request merged**